scripts/sites.js

```diff
@@ -766,7 +766,7 @@ const sites = [
     url: 'https://crlf.site',
     langs: ['en'],
     rss: 'https://crlf.site/feed.xml',
-    title: 'Cr;Lf;',
+    title: 'CrLf',
```

Comment:
haha, sorry, I wasn't clear enough. The title you had should be fine, but I meant the object keys. The `url` is out of order alphabetically to the other keys :)

Reply:
Yeah, sorry, it's totally different than what I thought I have to do :D 
Fixed now